tools/cuds-prioritized-attribution/test/tests/test_9_4/export.json

```diff
@@ -0,0 +1,268 @@
+{  
```

Comment:
nit: this is a lot of json files for all these test cases which is good to have coverage but do we need to keep these in github? could we have instructions to download theses test cases from a public GCS bucket?

Reply:
@jaketf There are files for test case. I will prefer to keep it in Github. Anything specific you are worried about. For me its better kept in one place. The following is my rational.

1. You don't need to switch between different places to checkout and run the code. 
2. When extending/changing, I (can be new user who does not have Google/PSO account) have to ask permission to GCS bucket to upload the verified test data.  
3. The maintenance extra bucket. 

Please let me know your rational behind this requested change.

Reply:
I don't feel strongly about this and you bring up solid points.
My thoughts just sprawl / size of this repo.
For better or worse, there are many codelabs / etc. that clone this repo (to use other examples / tools)
We essentially add ~1.2 MB of extra json files for said users to download (not the end of the world at all)
```bash
du -sh professional-services-1/tools/cuds-prioritized-attribution/test/tests/
1.2M	professional-services-1/tools/cuds-prioritized-attribution/test/tests/
```
I think it's fine to leave it in git, just floating the option.